examples/Atlas/+atlasControllers/QPInputConstantHeight.m

```diff
@@ -29,7 +29,7 @@
                         'y0', zeros(2,1),... % nominal output % 2 d
                         'u0', zeros(2,1),... % nominal input % 2 d
                         'R', zeros(2),... % input LQR cost % 4 d
-                        'Qy', eye(2),... % output LQR cost % 4 d
+                        'Qy', 0.8*eye(2),... % output LQR cost % 4 d
```

Comment:
This change is somewhat experimental, but seems to have improved walking reliability on Atlas. @kuindersma do you want to keep it?